File: app/src/main/java/my/game/loto/choiceAction/retrofit/ChoiceService.java
package my.game.loto.choiceAction.retrofit;

import java.util.List;

import my.game.loto.choiceAction.retrofit.settingsObjects.PlayObject;
import my.game.loto.choiceAction.retrofit.settingsObjects.StartingObject;
import retrofit2.http.Body;
import retrofit2.http.POST;
import rx.Observable;


public interface ChoiceService {

    @POST("getPlayObject")
    Observable<List<PlayObject>> getGame(@Body StartingObject startingObject );

}

File: app/src/main/java/my/game/loto/gameAction/retrofit/ChoiceService.kt
package my.game.loto.gameAction.retrofit

import my.game.loto.choiceAction.retrofit.settingsObjects.StartingObject
import my.game.loto.gameAction.retrofit.settingsObjects.PlayObject
import retrofit2.http.Body
import retrofit2.http.POST
import rx.Observable


interface ChoiceService {

    @POST("getPlayObject")
    fun getGame(@Body startingObject: StartingObject): Observable<List<PlayObject>>

}
